shared/my-sites/themes/theme-options.js

```diff
@@ -64,9 +64,15 @@ export default function getButtonOptions( site, theme, actions, setSelectedTheme
 	}
 };
 
-function rawOptions( site, theme ) {
+function rawOptions( site, theme, isLoggedOut ) {
 	return [
 		{
+			name: 'signup',
+			label: i18n.translate( 'Start a Blog' ),
```

Comment:
We want to avoid the word "blog" as it would exclude all our other users. We can iterate on that, but the latest proposal here is: 

> Register with this

Which is not perfect, but at least avoids the wording. :)

An alternative idea was:

> Choose this design